searchcorespi/src/vespa/searchcorespi/index/indexmaintainer.cpp

```diff
@@ -1020,14 +1025,18 @@ IndexMaintainer::runFusion(const FusionSpec &fusion_spec, std::shared_ptr<search
         serialNum = IndexReadUtilities::readSerialNum(lastFlushDir);
     }
     FusionRunner fusion_runner(_base_dir, args._schema, tuneFileAttributes, _ctx.getFileHeaderContext());
-    uint32_t new_fusion_id = fusion_runner.fuse(fusion_spec, serialNum, _operations, std::move(flush_token));
+    uint32_t new_fusion_id = fusion_runner.fuse(fusion_spec, serialNum, _operations, flush_token);
     bool ok = (new_fusion_id != 0);
     if (ok) {
         ok = IndexWriteUtilities::copySerialNumFile(getFlushDir(fusion_spec.flush_ids.back()),
                                                     getFusionDir(new_fusion_id));
     }
     if (!ok) {
-        LOG(error, "Fusion failed.");
+        if (flush_token->stop_requested()) {
+            LOG(info, "Fusion stopped.");
```

Comment:
Consider adding fusion dir (fail_dir) to message to give more context. Applies for message on line 1038 as well.